pymathics/natlang/main.py

```diff
@@ -349,7 +370,7 @@ def eval(self, text: String, evaluation: Evaluation, options: dict):
             return ListExpression(*[String(sent.text) for sent in doc.sents])
 
     def eval_n(self, text: String, n: Integer, evaluation: Evaluation, options: dict):
-        "TextSentences[text_String, n_Integer, OptionsPattern[%(name)s]]"
+        "TextSentences[text_String, n_Integer, OptionsPattern[TextSentences]]"
```

Comment:
Why this change?

Reply:
The short answer is clarity over cleverness.

There are levels of rules or conventions going on here.

The first convention is Mathics3 Case pattern matching which is used in rule application to figure out which class method to call. Stuff like `_String` fall into that category. 

And the second convention has to do with Python docstrings. and `%(name)s` falls into this category. 

`%(name)s` is not that commonly used and I don't want people to confuse this for a Mathics3 kind of pattern. 

This cross-level confusion we see in the homegrown docstring doctests as well, where `\` before a newline is a Python string thing, but it is embedded within the homegrown doc system. 

`%(name)s` is more suited when there is a changes the class or may change and we want to adjust this automatically.  However the chance that `TextSentence` will change is not likely as it is a WMA builtin. And if it does, it probably good to go over everything explicitly. 

Again, in short we have had too much cleverness and not a lot of clarity. I think we should try clarity first, and cleverness when it is really warranted which is not here.

Reply:
Actually, the argument in `OptionsPattern` is not needed here.

Reply:
Ok - after this lands, if you are up for it  would you go in and remove all the the `OptionsPattern`s that are not needed? 

Also if you are up for it remove the `to_expression`s and `to_lists` by adding the additional conversions where needed.